workflows/real_time_ai_surgical_video_processing/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.20)
+project(real_time_ai_surgical_video_processing_workflow LANGUAGES NONE)
+
+# Create a custom target that runs the Python script
+add_custom_target(real_time_ai_surgical_video_processing
+    COMMAND Python3::Interpreter ${CMAKE_CURRENT_SOURCE_DIR}/python/main.py
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+)
+
+# Make sure Python interpreter is found
+find_package(Python3 COMPONENTS Interpreter REQUIRED)
+
```

Comment:
Why do we need this?

Reply:
@jjomier doesn't it essentially check if the python exisits? Do you think it is redundant?

Reply:
I just don't understand the target.

Reply:
Oh right! I thought you were taking about the find_package part. I added that before talking to you for enabling the data downloads, which didn't work. Thanks for spotting it, I remove it.